agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -609,6 +609,48 @@ func TestMetadataServiceValidator(t *testing.T) {
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
 
+// TestAgentIntrospectionValidator tests that the agent introspection endpoint can
+// be accessed from within the container.
+func TestAgentIntrospectionValidator(t *testing.T) {
+	// The Docker version was 17.03.1-ce when we added changes to agent
+	// introspection endpoint feature for the last time.
+	RequireDockerVersion(t, ">=17.03.1-ce")
```

Comment:
Why was this required? The introspection API shouldn't have dependency on the docker version.